pkg/config/agent/config.go

```diff
@@ -202,9 +202,10 @@ type AgentConfig struct {
 }
 
 type AntreaProxyConfig struct {
+	// To enable AntreaProxy, you need to set "enable" to true.
+	Enable bool `yaml:"enable,omitempty"`
```

Comment:
I think it may better to make this a pointer (`*bool`) so we can default it to true. This way, if someone forgets to update their config, it will still behave as expected.

Reply:
Done